<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>345</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="ReadLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить колонку: укажите название</string>
   </property>
  </widget>
  <widget class="QLabel" name="ReadLabel_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить колонку: укажите название</string>
   </property>
  </widget>
  <widget class="QLabel" name="ReadLabel_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Rename: формат Старая колонка-Новая колонка</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditDelete">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditRename">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="DeleteColumnsButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditNewColumn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="CreateNewColumnButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>130</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Create New Column</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RenameColumnButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>230</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Rename Column</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DropAllRowsButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>440</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Дропнуть весь источник</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditDeleteReport">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="DeleteReportButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>320</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete Report</string>
   </property>
  </widget>
  <widget class="QLabel" name="ReadLabel_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить отчет: укажите номер отчета</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
